client/app/templates/components/item-detail-page.hbs

```diff
@@ -1,37 +1,60 @@
         <div class="row">
-            <div class="col-xs-6 text-left">  {{#link-to 'collection.group.item' item.previousId}}<i class="fa fa-chevron-left"></i> Previous {{/link-to}}</div>
-            <div class="col-xs-6 text-right">{{#link-to 'collection.group.item' item.nextId}}Next <i class="fa fa-chevron-right"></i> {{/link-to}}</div>
+            <div class="col-xs-12 col-sm-10">
+                <h1 class="p-b-md">{{item.title}}</h1>
+                <h5 class="view-authors">
+                    <ul class="comma-list">
+                        {{#if authors}}
+                            {{#each authors as |author index| ~}}
+                                {{#if author.bibliographic}}
+                                    <li><a href={{author.users.profileURL}} {{action 'click' 'link' 'Content - Author' author.users.profileURL}}>{{author.users.givenName}} {{author.users.familyName}}</a></li>
+                                {{/if}}
+                            {{~/each}}
+                        {{else}}
+                            <div class="loader">
+                                <div class="loader-inner ball-pulse">
+                                  <div></div>
+                                  <div></div>
+                                  <div></div>
+                                </div>
+                            </div>
+                        {{/if}}
+                    </ul>
+                </h5>
+                <p>Added on: {{moment-format model.datePublished "MMMM DD, YYYY"}} | Last edited: {{moment-format model.dateModified "MMMM DD, YYYY"}} </p>
+            </div>
         </div>
     </div>
 </div>
-<div class="coll-item-page">
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-4">
-                <h2> {{item.title}}</h2>
-                <p> {{item.description}}</p>
-                <div>
-                    {{#each item.tags as |tag|}}
-                        <span class="coll-item-tag">{{tag}}</span>
-                    {{/each}}
-                </div>
-                <div>Status: {{item.status}}</div>
-                <div class="m-t-xl">
-                    <button class="btn btn-default">Remove from collection </button>
-                </div>
+<div class="container">
+    <div class="row p-t-lg">
+        <div class="col-sm-8 {{unless item.hasWiki 'hidden'}}">
+            <div>
+                <pre style='white-space: pre-wrap; border:none; width:100%; text-align:justify; max-height: 300px;'>
+                    {{wikiContent}}
+                </pre>
             </div>
-            <div class="col-sm-8">
-                <div class="coll-mock-mfr">
+        </div>
+        <div class="col-sm-8 {{unless item.hasFile 'hidden'}}">
+            <div class="coll-mock-mfr">
+                {{file-renderer download="https://staging-files.osf.io/v1/resources/2t6qv/providers/osfstorage/58a4c6638ca57e01fddf0a14" width="99%" height="1000" allowfullscreen=true}}
```

Comment:
This project must be private because I'm getting a 403 error from MFR. Also, is there a way to show the actual preprint file?